README.md

```diff
@@ -0,0 +1,35 @@
+# Remix Compute for Netlify
+
+The Remix Compute for Netlify packages allows you to deploy your Remix apps to Netlify on the Edge.
+
+This project is not a template for Remix apps. It is a set of packages used by the Netlify Remix template.
+
+The project is a monorepo. Packages are located in the `packages` directory. There are three packages:
+
+- `@netlify/remix-runtime` - The Remix runtime for Netlify on the Edge
```

Comment:
The product is "Netlify Edge Functions"

Reply:
Yeah, I wasn't sure if that applied to our edge offering as a whole. Thanks. I'll update it.